package.json

```
@@ -80,6 +81,7 @@
     "react-datepicker": "^4.13.0",
     "react-dom": "17.0.2",
     "react-helmet": "^6.1.0",
+    "react-instantsearch": "7.7.0",
```

Allow minor versions updates
```suggestion
    "react-instantsearch": "^7.7.0",
```